src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessTokenClient/Altinn.Common.AccessTokenClient.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <OutputType>Library</OutputType>
+    <Version>0.5.0</Version>
+    <Description>Package to generate access tokens to use against Altinn Platform.  Can be used by applications and other components that need to access</Description>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Company>Altinn</Company>
+    <RepositoryUrl>https://github.com/Altinn/altinn-studio</RepositoryUrl>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
```

Comment:
Consider adding StyleCop to this project as well.

Reply:
Done